spec/services/contributors_parser_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe "ContributorsParser" do
+  it "should create contributors for a legislator from an api response" do
+    legislator = create(:legislator)
+
+    response = [ { "employee_amount": "1504133.00",
```

Comment:
Maybe extract a `build_contributor` method that takes a hash of attributes you can override.

So you can do 
```ruby
response = [build_contributor(name: "Univ. of Cali."),
                    build_contributor(name: "Microsoft")]
```